### ISSUE TITLE
train loss NaN !!

### ISSUE THREAD

**Issue by 9B8DY6** [rocketappslab/bdmm#5]
I run train.py with your code but I ve got Nan in loss graph. I did not change dataset and any setting. 
![image](https://github.com/rocketappslab/BDMM/assets/67573223/e76b047f-a4dc-4755-8355-527de7e49bc6)

**Comment by YangShenglong-ai** [comment]
I had the same problem. How did you solve it?

**Comment by YangShenglong-ai** [comment]
发现在莫一时刻，cx_gen loss 变成了nan，修改CX_style_loss.py 文件 def calc_relative_distances, 最后添加 relative_dist= torch.clamp(relative_dist, -1, 1, )虽然是强行限制，但应该不会报错了。希望作者可以给出更加有效的解决方法
